RecoTracker/IterativeTracking/python/HighPtTripletStep_cff.py

```diff
@@ -32,6 +32,25 @@
     FPix = dict(skipClusters = cms.InputTag('highPtTripletStepClusters'))
 )
 
+eras.trackingPhase2PU140.toModify(highPtTripletStepSeedLayers, 
+    layerList = ['BPix1+BPix2+BPix3', 'BPix2+BPix3+BPix4',
+                 'BPix1+BPix3+BPix4', 'BPix1+BPix2+BPix4',
+                 'BPix2+BPix3+FPix1_pos', 'BPix2+BPix3+FPix1_neg',
+                 'BPix1+BPix2+FPix1_pos', 'BPix1+BPix2+FPix1_neg',
+                 'BPix2+FPix1_pos+FPix2_pos', 'BPix2+FPix1_neg+FPix2_neg',
+                 'BPix1+FPix1_pos+FPix2_pos', 'BPix1+FPix1_neg+FPix2_neg',
+                 'BPix1+BPix2+FPix2_pos', 'BPix1+BPix2+FPix2_neg',
+                 'FPix1_pos+FPix2_pos+FPix3_pos', 'FPix1_neg+FPix2_neg+FPix3_neg',
+                 'BPix1+FPix2_pos+FPix3_pos', 'BPix1+FPix2_neg+FPix3_neg',
+                 'BPix1+FPix1_pos+FPix3_pos', 'BPix1+FPix1_neg+FPix3_neg',
+                 'FPix2_pos+FPix3_pos+FPix4_pos', 'FPix2_neg+FPix3_neg+FPix4_neg',
+                 'FPix3_pos+FPix4_pos+FPix5_pos', 'FPix3_neg+FPix4_neg+FPix5_neg',
+                 'FPix4_pos+FPix5_pos+FPix6_pos', 'FPix4_neg+FPix5_neg+FPix6_neg',
+                 'FPix5_pos+FPix6_pos+FPix7_pos', 'FPix5_neg+FPix6_neg+FPix7_neg',
+                 'FPix6_pos+FPix7_pos+FPix8_pos', 'FPix6_neg+FPix7_neg+FPix8_neg',
+                 'FPix6_pos+FPix7_pos+FPix9_pos', 'FPix6_neg+FPix7_neg+FPix9_neg']
+)
```

Comment:
Would it be more clear if the phase2 layer list would be specified as an extension to phase1 list? (just asking, I'm not sure)